tests/integration/go_ethereum/test_goethereum_http.py

```diff
@@ -129,6 +134,10 @@ def test_admin_start_stop_ws(self, web3: "Web3") -> None:
         super().test_admin_start_stop_ws(web3)
 
 
+class TestGoEthereumAsyncAdminModuleTest(GoEthereumAsyncAdminModuleTest):
+    pass
```

Comment:
You can copy the tests w/ xfails from the above `TestGoEthereumAdminModuleTest` class into here

Reply:
copied the test and modified for async